src/frontend/context/PlaylistContext.js

```diff
@@ -0,0 +1,93 @@
+import { createContext, useContext, useReducer, useEffect } from 'react';
+import { useAuthCtx } from './index';
+import { getPlaylists, addPlaylist, deletePlaylist } from '../service';
+import { useLocalStorage } from '../helpers';
+// import { historyReducerFunction, historyDefaultState } from '../helpers';
```

Comment:
remove comments if not in use

Reply:
that's a good suggestion. I will put empty check also